public/app/features/explore/RichHistory/RichHistoryCard.tsx

```diff
@@ -123,12 +131,18 @@ export function RichHistoryCard(props: Props) {
             <Forms.TextArea
               value={comment}
               placeholder={comment ? undefined : 'add comment'}
-              onChange={e => setComment(e.currentTarget.value)}
+              onChange={e => {
+                setComment(e.currentTarget.value);
+              }}
+              onClick={e => {
+                e.stopPropagation();
+              }}
```

Comment:
Add stop propagation, as clicking on comment and comment buttons also triggered adding queries to query fields.

Reply:
This seems weird is it a bug in the form elements?